src/views/Home/Home.test.jsx

```diff
@@ -8,8 +12,47 @@ const user = {
   likes: ['React', 'Anime', 'Traveling', 'Living', 'Tower Defense Games', 'Card Games'],
   motto: 'Res Non Verba',
   color: 'crimson',
-}
+};
 
-test('Should render the user profile', () => {
+describe('Profile Tests', () => {
+  test('Should render the user profile', async () => {
+    render(
+      <MemoryRouter>
+        <Home user={user} />
+      </MemoryRouter>
+    );
 
-})
+    // screen.debug();
+
+    // const heading = screen.getByRole('heading');
+    // expect(heading.textContent).toEqual('Vonta');
+
+    // check for profile name
+    await waitFor(() => {
+      const profile = screen.getByText(/vonta/i);
+      expect(profile.textContent).toBe('Vonta');
+    });
+
+    // check for motto
+    await waitFor(() => {
+      const motto = screen.getByText(/res non verba/i);
+      expect(motto.textContent).toBe('Res Non Verba');
+    });
+
+    // check for interests heading
+    waitFor(() => {
```

Comment:
While these `waitFor`'s work, they can be repetitive to read through.  Consider using `await screen.findBy` instead of wrapping `getBy` in a `waitFor` block.